.github/workflows/build-mingw64.yml

```
@@ -0,0 +1,60 @@
+name: 'mingw64 release Builds'
+on:
+  workflow_call:
+    inputs:
+      do_package:
+        required: false
+        type: boolean
+        default: false
+      do_package_symbols:
+        required: false
+        type: boolean
+        default: false
+      retention_days:
+        required: false
+        type: number
+        default: 14
+      run_build:
+        required: true
+        type: boolean
+
+jobs:
+  gcc13-release:
+    name: GCC 13.2 release Builds for mingw64
+    environment: public-github-runners
+    runs-on: windows-latest
+    if: inputs.run_build == true
+    steps:
+      - name: setup
+        run: |
+          echo "C:\msys64" >> $GITHUB_PATH
+          echo "C:\msys64\mingw64\bin\" >> $GITHUB_PATH
+      - name: Install Dependencies
+        shell:  C:\shells\msys2bash.cmd {0}
+        run: |
+          pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja --noconfirm
+          export PATH="/mingw64/bin/:$PATH"
```

Alternatively, move the `shell` above `run` in the other steps.

```suggestion
        run: |
          pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja --noconfirm
          export PATH="/mingw64/bin/:$PATH"
        shell:  C:\shells\msys2bash.cmd {0}
```